index.rst

```diff
@@ -122,6 +124,29 @@ Indices
 - :ref:`genindex`
 - :ref:`Search <search>`
 
+.. _part-usage:
+
+Use of Rubin Observatory Software
+=================================
+
+All LSST Science Pipelines code is `free software <http://www.gnu.org/philosophy/free-sw.html>`_, licensed under the terms of the `GNU General Public Licence, Version 3 <http://www.gnu.org/copyleft/gpl.html>`_.
+You have the freedom to run, copy, distribute, study, change and improve the software as you see fit, with no strings attached (aside from the GPL requirements when redistributing the code or derivatives).
```

Comment:
I worry that the "no strings attached" language is at odds with the parenthetical statement. Perhaps this is more accurate?

```suggestion
You have the freedom to run, copy, distribute, study, change and improve the software as you see fit within the terms of the GPL v3 license.
```

Do you think it'd be beneficial to use the [Choose a license page](https://choosealicense.com/licenses/gpl-3.0/) to provide clear guidelines to users on what the license means?

Reply:
Happy to change the text as you suggest. What would be your specific recommendation with regards the 'Choose a license page'?

Reply:
The choosealicense.com provides executive summaries of different software licenses (it was created by GitHub). It may be overreach here, so we can skip it.